jws.go

```diff
@@ -13,10 +13,12 @@ package goacme
 
 import (
 	"crypto/rsa"
+	"crypto/sha256"
 	"encoding/base64"
 	"encoding/json"
-
+	"fmt"
 	jose "github.com/letsencrypt/go-jose"
```

Comment:
can you separate `jose` import from everything else. normally, you'd have something like this:

```go
import (
  "stdlib/pkg1"
  "stdlib/pkg2"

  "github.com/xxx"
)
```

In other words, stdlib imports are separated from external packages with an empty line.